### DISCUSSION TITLE
Loop through generations while slightly changing the prompts and settings

### DISCUSSION THREAD

**Discussion by tjthejuggler** [Comfy-Org/ComfyUI#1193] [General]
Can anyone tell me if this is something that ComfyUI can do?

Example Prompt: A **color** **animal**

I want to have 100 images, each with their number in the name. Each set of 10 I want to have a different color from a list I provide. The animal I want to be chosen randomly from a list I provide for each image. Every 20 images I want to increase the number of steps by 5. Is all of this doable with ComfyUI?

Thanks so much!

**Comment by ltdrdata** [comment]
By utilizing math nodes, wildcards, and an infinite queue, you can accomplish this task.

**Reply by tjthejuggler** [reply]
Thanks so much! You've given me hope!

**Reply by tjthejuggler** [reply]
I've been unable to find any default math nodes, where these custom ones the ones you were referring to?

https://github.com/Derfuu/Derfuu_ComfyUI_ModdedNodes

**Comment by ltdrdata** [comment]
> I've been unable to find any default math nodes, where these custom ones the ones you were referring to?
> 
> https://github.com/Derfuu/Derfuu_ComfyUI_ModdedNodes

It's correct that vanilla doesn't provide math nodes. There are several custom nodes available that offer math functionalities, so you can choose the one that suits you best for convenience.

**Reply by tjthejuggler** [reply]
Alrighty, thanks so much, I appreciate you!
